tests/integration/test_counts.py

```diff
@@ -0,0 +1,115 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from tests.integration import test_submit, utility_test_functions
+
+workflows_path = Path(__file__).parents[1].joinpath("workflows/")
+
+
+def check_status(test_workflow_id: str):
+    import time
+
+    count = 0
+    status = ""
+    print("Printing workflow status:")
+    while count < 10:
+        time.sleep(5)
+        count = 1 + count
+        # run status 1 time a minute unitl done, max 5x
```

Comment:
I'm not quite understanding this comment.

Reply:
edited to be "run a workflow status check every 5sec until done, max 10x"